galleries/examples/text_labels_and_annotations/fancyarrow_demo.py

```diff
@@ -21,29 +25,29 @@
        .add_gridspec(1 + nrow, ncol,
                      wspace=0, hspace=0, left=0, right=1, bottom=0, top=1).subplots())
```

Comment:
```
spacing = {'wspace': 0, 'hspace': 0.05, 'left': 0, 'right': 1, 'bottom': 0, 'top': 1}
fig, axs = plt.subplots(1 + nrow, ncol, figsize=(4 * ncol, 1 + nrow), gridspec_kw=spacing)
```

Though these don't even need to be lined up in grid like this, and could conceivably be two subfigures side by side.